src/test/java/com/rakovets/course/java/core/practice/io/FileAnalyzeUtilTest.java

```diff
@@ -0,0 +1,96 @@
+package com.rakovets.course.java.core.practice.io;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class FileAnalyzeUtilTest {
+
+    @Test
+    void getListOfStrings() {
+        String expectedList = "[Aux test out time enum, compile under Inbox, Aux test out time enum, Aux test]";
+
+        String actualList = FileAnalyzeUtil.getListOfStrings("src\\test\\java\\com\\rakovets\\course\\java\\core\\practice\\io\\text.txt").toString();
```

Comment:
This is Windows way. This is very bad practice. Use path separator.

You have broked tests.